File: week02/day04/src/Reverse.java
/**
 * Created by Bálint on 2017. 03. 23..
 */
public class Reverse {
  public static void main(String... args){
    String reversed = ".eslaf eb t'ndluow ecnetnes siht ,dehctiws erew eslaf dna eurt fo sgninaem eht fI";

    // Create a function that can reverse a String, which is passed as the parameter
    // Use it on this reversed string to check it!
    // Try to solve this using charAt() first, and optionally anything else after.
    StringBuilder good = new StringBuilder(reversed);
    System.out.println(good.reverse());
  }
}

File: javascript_practice/3-data-structures/reverse.js
'use strict';
// Reverse the string!

var reversed = ".eslaf eb t'ndluow ecnetnes siht ,dehctiws erew eslaf dna eurt fo sgninaem eht fI"
var correctOrder = '';

while (reversed.charAt(0) !== '') {
  correctOrder = correctOrder + reversed.charAt(reversed.length - 1);
  reversed = reversed.slice(0, reversed.length - 1);
}
reversed = correctOrder;

console.log(reversed);
